Codecademy/gamut
# feat(ColorMode): correct background-current from useColorMode hook

### Overview


adds correct `background-current` to useColorMode hook + adjusts the mobile GlobalHeader to match these [designs](https://www.figma.com/file/40NSFXquATvE19Gm9pnJzI/Catalog-navigation-dropdown?node-id=1080%3A18028) 


### PR Checklist

- [x] Related to JIRA ticket:[ [ABC-123]](https://www.figma.com/file/40NSFXquATvE19Gm9pnJzI/Catalog-navigation-dropdown?node-id=1080%3A18028)

#### Testing Instructions



* Go to the GlobalHeader page in Storybook
* Nav to "Free User - Default" (the first example on the page)
* When the mobile menu is open, the headerbar should be beige (like so)
* 
<img width="467" alt="image" src="https://user-images.githubusercontent.com/58054751/193115265-01703857-caae-4f3c-850d-8027aa3dc6bd.png">

* Go to each of the respective PR envs + follow each of the testing instructions there 

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/32783)  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/299)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-299)   |
